src/library/scala/collection/concurrent/TrieMap.scala

```diff
@@ -912,6 +913,18 @@ extends scala.collection.concurrent.Map[K, V]
       }
     }
   }
+  def fail[K, B](self: TrieMap[K, V]): Option[V] = {
```

Comment:
Looks good to me overall, but I failed to understand what `fail` is for. And why does it have a `B` type parameter?